cmd/crossplane/main.go

```diff
@@ -36,8 +36,9 @@ func main() {
 	var (
 		log = logging.Logger
 
-		app   = kingpin.New(filepath.Base(os.Args[0]), "An open source multicloud control plane.").DefaultEnvars()
-		debug = app.Flag("debug", "Run with debug logging.").Short('d').Bool()
+		app         = kingpin.New(filepath.Base(os.Args[0]), "An open source multicloud control plane.").DefaultEnvars()
+		debug       = app.Flag("debug", "Run with debug logging.").Short('d').Bool()
+		syncMinutes = app.Flag("sync", "Controller manager sync period in minutes").Short('s').Default("60").Uint()
```

Comment:
I did not review this PR in time, but I think it would be preferable to use `Duration` instead of `Uint`. This allows more flexibility in specifying the time -  `1h`, `30m`, `20s`, etc.

Reply:
Ah.. good point. I will update in another PR